code/modules/client/preference/character.dm

```diff
@@ -108,6 +108,8 @@
 	var/list/custom_emotes = list()
 	/// Runechat color
 	var/runechat_color = "#FFFFFF"
+	/// The ringtone their PDA should start with
+	var/pda_ringtone = "beep"
```

Comment:
This should be null by default so that job-specific ringtones are still applied if you dont have a preference

Reply:
It also needs some way to set the ringtone to null so you can go back the default